src/components/EnterPinModal.tsx

```diff
@@ -47,18 +55,16 @@ export default class EnterPinModal extends React.Component<Props, State> {
           pin: '',
           submitting: false,
          });
-        // Prompt another pin so that user can try again  
-        // Annoying that typescript makes me check this ...
-        if (device) {
```

Comment:
`promptPin` no longer targets a specific device. It sends pin requests to every plugged in device with `needs_pin_sent`.  This made UI more intuitive to use.